index.html

```diff
@@ -5025,6 +5063,64 @@ <h3>Device Theft and Impersonation</h3>
 All or any combination of the above.
           </li>
         </ul>
+
+        <p>
+Furthermore, instances of impersonation can manifest in various forms,
+including situations where an entity attempts to disavow their actions. 
+Elevating the level of trust and security within the realm of <a>verifiable credentials</a>
+entails more than just averting impersonation; it involves the implementation
+of non-repudiation mechanisms. These mechanisms solidify an entity's responsibility
+for their actions or transactions, thereby reinforcing accountability and deterring
+malicious behaviors. The attainment of non-repudiation is a multifaceted endeavor,
+encompassing an array of techniques ranging from 
+<a href=#securing-verifiable-credentials>securing mechanisms</a>, proofs of possession,
+and authentication schemes to various protocols designed to foster trust and reliability.
+        </p>
+      </section>
+      <section class="informative">
+        <h3>Acceptable Use</h3>
+
+        <p>
+Ensuring that there is alignment between an entity's actions, such as <a>presentation</a>,
+and the intended purpose of those actions, is of paramount importance. It involves
```

Comment:
```suggestion
and the intended purpose of those actions, is of importance. It involves
```

Not sure if it's of "paramount" importance since it's hard to gauge intended purpose in a remote setting.